### DISCUSSION TITLE
Limit transforms to assets?

### DISCUSSION THREAD

**Discussion by jhultgre** [Autodesk/maya-usd#1077] [Q&A]
Is there a way to setup a stage so artists can only select and transform things on the asset level?
Right now as I've been playing around with it I can select any individual mesh of an asset and move them independently, which I don't want.

I'm trying to imitate our existing pipeline where that is a limitation.

**Comment by kxl-adsk** [comment]
With VP2RenderDelegate and UFE selection you can use `UsdSelectionKind` option var to control what gets selected after picking. Have a look at the PR description for more details https://github.com/Autodesk/maya-usd/pull/641.
